src/map/clif.c

```diff
@@ -20377,6 +20381,19 @@ static void clif_monster_hp_bar(struct mob_data *md, struct map_session_data *sd
 #endif
 }
 
+static bool clif_show_monster_hp_bar(const struct mob_data *md, struct block_list *bl) {
```

Comment:
This function should take only `struct block_list *` as an argument and get `struct mob_data *` locally, there is no point in taking both.

Reply:
Please see changes in latest commit